### PR TITLE
Add Bilibili

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: entries/b/bilibili.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Bilibili": {
+    "domain": "bilibili.com",
+    "contact": {
+      "facebook": "bilibilidouga",
```
Comment (Carlgo11) [inline entries/b/bilibili.com.json]: As the CircleCI build says, the Facebook handle `bilibilidouga` is either private or doesn't exist. Please remove it.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Your PR contains changes for `bilibili.com`, `bitflyer.com` and `paypay.ne.jp`. Please remove the commits not related to Bilibili.

File: entries/b/bilibili.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Bilibili": {
+    "domain": "bilibili.com",
+    "contact": {
+      "twitter": "bilibili_en",
+      "language": "en"
```
Comment (Carlgo11) [inline entries/b/bilibili.com.json]: The default language is English
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/b/bilibili.com.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...Zm-70.91-22.79q37.67,0,75.32,0a11.63,11.63,0,0,0,6.18-1.44c5.22-3.14,7.51-7.94,7.52-13.87q0-48.56,0-97.12a16.79,16.79,0,0,0-.64-5.14c-2-6.25-6.52-10.57-13.5-10.57q-75,0-149.94,0a11.87,11.87,0,0,0-7,2.15A15.72,15.72,0,0,0,60.9,111c-.07,31.92,0,63.84,0,95.75a32.28,32.28,0,0,0,.4,4.49,14.54,14.54,0,0,0,15,12.5C100.73,223.4,125.25,223.61,149.75,223.61Z"/><path class="cls-1" d="M220.94,155.89l-47.5-9.17c1.18-6.07,2.33-12,3.52-18.18,15.8,3.22,31.68,6,47.5,9.33C223.24,144.1,222.12,149.86,220.94,155.89Z"/><path class="cls-1" d="M149.9,192.59s-1.6,1.32-2.37,2c-6.4,5.76-15.2,5.35-21.85-.22-2-1.67-5.52-5.55-5.52-5.55s-1.2-1.27-1.25-1.36c2.2-1.39,5.66-3.54,7.84-4.93l1.3,1.36a20.08,20.08,0,0,0,5.69,4.89,6.75,6.75,0,0,0,7.76-1c3-2.6,4.89-6,7-9.27a16.94,16.94,0,0,1,1.43-1.85s2,2.67,2.84,4c1.71,2.7,3.36,5.45,6,7.4a6.67,6.67,0,0,0,8.14.25,22.22,22.22,0,0,0,4.48-4c.48-.54,1.75-1.75,1.74-1.75s4.9,2.95,7.28,4.67a8.89,8.89,0,0,1-2,3,23.16,23.16,0,0,1-10,7.48c-6.22,2.26-11.74.79-16.61-3.53-.51-.46-1.42-1.16-1.93-1.62C149.88,192.58,149.9,192.6,149.9,192.59Z"/><path class="cls-1" d="M79.78,156l47.5-9.17c-1.17-6.07-2.32-12.05-3.51-18.19-15.8,3.23-31.68,6-47.5,9.33C77.48,144.22,78.61,150,79.78,156Z"/></svg>
```
Comment (Carlgo11) [inline img/b/bilibili.com.svg]: From the logo on their website:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 16">
  <path fill="#00a1d6" d="M13.57 0a.95.95 0 0 0-.74.3l-2.12 2.03a.78.78 0 0 0-.15.2H7.35a.98.98 0 0 0-.16-.2L5.07.3C4.13-.6 2.78.8 3.71 1.7l.85.82h-.58a4 4 0 0 0-4 4V12a4 4 0 0 0 4 3.99h10a4 4 0 0 0 4-4V6.52a4 4 0 0 0-4-3.99h-.64l.85-.82c.71-.68.12-1.64-.62-1.7Zm-9.6 4.45h10a2 2 0 0 1 2 2v5.63a2 2 0 0 1-2 2h-10a2 2 0 0 1-2-2V6.44c0-1.1.9-2 2-2zM12 7.12a1 1 0 0 0-1 1v.9c0 1.33 2 1.33 2 0v-.9a1 1 0 0 0-1-1Zm-6 0c-.5 0-1 .33-1 1v.9c0 1.32 2 1.32 2 0v-.9c0-.66-.5-1-1-1Z"/>
</svg>
```

File: entries/b/bilibili.com.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    },
+    "keywords": [
+      "entertainment"
+    ]
```
Comment (Carlgo11) [inline entries/b/bilibili.com.json:12]: As the website is entirely in Chinese I think specifying a regions element would be useful.
```suggestion
    ],
    "regions": [
      "cn"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/b/bilibili.com.json (hunk 4 above)
Comment (ghost) [reply entries/b/bilibili.com.json:12]: The website is written in Chinese, but you can sign up from all over the world. In fact, Bilibili has many users in Japan. Language and region are two different concepts.

Comment (Carlgo11) [reply entries/b/bilibili.com.json:12]: The regions element isn't meant to list regions where it's technically possible to use the site but the intended market audience of a site. If the creators of Bilibili had the ambition of reaching a global audience I'm pretty sure they would have added multi-language support.
E.g. [taobao.com](taobao.com) and [aliexpress.com](https://www.aliexpress.com/).

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**